github/ci/prow-deploy/vars/kubevirtci-testing/secrets.yml

```diff
@@ -28,6 +28,7 @@ installerPullToken: '{}'
 coverallsToken: d3c481176ace1b6c5eb417b0d3dd01497
 
 bootstrap_full_config: true
+reconcile_github_webhooks: false
```

Comment:
We might set this to true to test it against a test repo? Needs mock config then though.